redwood-treehouse-host/src/commonMain/kotlin/app/cash/redwood/treehouse/TreehouseContentSource.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.treehouse
+
+public fun interface TreehouseContentSource<A : AppService> {
```

Comment:
I get we didn't have any kdoc for this before, but would it be possible to add some? Whenever I dig through Redwood source code, any and all kdoc is useful to wrap my head around why stuff is done a certain way.